src/sticky.js

```diff
@@ -94,15 +94,30 @@ export default class Sticky extends React.Component {
     });
   }
 
-  recomputeState = () => {
+	isInView() {
+		const doc = document.documentElement
+		const portTop = (window.pageYOffset || doc.scrollTop) - (doc.clientTop || 0)
+		const portBottom = portTop + window.innerHeight
+
+		const node = ReactDOM.findDOMNode(this)
+		const nodeTop = node.offsetTop
+		const nodeBottom = nodeTop + node.getBoundingClientRect().height
+
+		return nodeBottom >= portTop && nodeTop <= portBottom
+			|| !!node.querySelector('.' + this.props.stickyClassName)
```

Comment:
Is there a reason you are using `node.querySelector(...)` instead of just looking at `this.state.isSticky`? I would prefer the latter unless there is a specific reason not to.